### PR TITLE
Remove last remains of cross referencing

### PR DESCRIPTION
**Pull request by larswmh**
In [my latest pull request](https://github.com/corona-warn-app/cwa-website/pull/1685) I've removed cross referencing by arrows and started to use links instead. However, I ended up missing one (thanks to @MikeMcC399 for telling me). This pull request will remove the last remains of cross referencing arrows. See the images below

Right now: 
chrome_Em2sOGhmuc

After merging this pull request: 
chrome_SkBIga4gQr

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -2148,7 +2148,7 @@
                 {
                     "term": "Impfzertifikat",
                     "anchor": "vaccination_certificate",
-                    "description": "Das Impfzertifikat ist ein => <a href='#glossary_digital_certificate'>digitales Zertifikat</a>, das einer Person ausgestellt wird, die eine Impfung mit einem in der EU zugelassenen Impfstoff wahrgenommen haben. Jedes Impfzertifikat dokumentiert genau einen Impfvorgang (eine Dosis). <br/><br/> Das Impfzertifikat stellt eine digitale Form für einen => <a href='#glossary_vaccination_proof'>Impfnachweis</a> dar."
+                    "description": "Das Impfzertifikat ist ein <a href='#glossary_digital_certificate'>digitales Zertifikat</a>, das einer Person ausgestellt wird, die eine Impfung mit einem in der EU zugelassenen Impfstoff wahrgenommen haben. Jedes Impfzertifikat dokumentiert genau einen Impfvorgang (eine Dosis). <br/><br/> Das Impfzertifikat stellt eine digitale Form für einen => <a href='#glossary_vaccination_proof'>Impfnachweis</a> dar."
```
Comment (mohe2015) [inline src/data/faq_de.json:2151]:
```suggestion
                    "description": "Das Impfzertifikat ist ein <a href='#glossary_digital_certificate'>digitales Zertifikat</a>, das einer Person ausgestellt wird, die eine Impfung mit einem in der EU zugelassenen Impfstoff wahrgenommen haben. Jedes Impfzertifikat dokumentiert genau einen Impfvorgang (eine Dosis). <br/><br/> Das Impfzertifikat stellt eine digitale Form für einen <a href='#glossary_vaccination_proof'>Impfnachweis</a> dar."
```
or am I stupid?

Comment (larswmh) [reply src/data/faq_de.json:2151]: You're right, thanks!

**Review by MikeMcC399 — APPROVED** [review]
LGTM!
No more occurrences of `=>` found in `faq.json` and `faq_de.json`

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@larswmh @MikeMcC399 @mohe2015 Thanks everybody.

**Pull request merged**